replication-api-impl/src/test/java/com/connexta/replication/api/impl/data/ReplicationItemManagerTest.java

```diff
@@ -85,19 +84,20 @@ public void getItemsForConfig() {
         .thenReturn(
             new PageImpl(
                 items.stream().map(ReplicationItemImpl.class::cast).collect(Collectors.toList())));
-    assertThat(itemManager.getItemsForConfig("configId", 0, 1), is(items));
-  }
-
-  @Test
-  public void saveItem() {
-    ReplicationItemImpl item = new ReplicationItemImpl();
-    itemManager.saveItem(item);
-    verify(itemRepository).save(eq(item));
+    assertThat(itemManager.getAllForConfig("configId", 0, 1), is(items));
   }
 
+  // BOB: redefine this one
```

Comment:
Should this just be removed or are you coming back to it?

Reply:
I am coming back to it as part of fixing and adding unit tests. `BOB` is my personal `TODO` :-)

Reply:
Removed